### PR TITLE
major changes to user context menu

### PR DESCRIPTION
**Pull request by skruglov2023**
Major change in context menu, shouldn't be breaking. Added a new option (onteam) that integrates with the existing onteam functionality to show all teams a user is on

### CODE REVIEW

File: dozer/cogs/profile_menus.py [hunk 1]
```diff
@@ -10,42 +16,92 @@ class ProfileMenus(commands.Cog):
     """
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
-        self.ctx_menu = app_commands.ContextMenu(
-            name = 'View Profile',
-            callback = self.profile,  # sets the callback the view_profile function
-        )
-        self.bot.tree.add_command(self.ctx_menu)  # add the context menu to the tree
-
-    async def cog_unload(self) -> None:
```
Comment (tweirtx) [inline dozer/cogs/profile_menus.py:19]: Why was this deleted?

Comment (skruglov2023) [reply dozer/cogs/profile_menus.py:19]: This one only allowed one context menu per class/cog, updated version allows for multiple commands. It also makes more sense snice the format is similar to regular commands

**Review by tweirtx — APPROVED** [review]
